components/MapView.js

```diff
@@ -413,14 +442,14 @@ class MapView extends React.Component {
       case 'android':
         NativeModules.UIManager.dispatchViewManagerCommand(
           this._getHandle(),
-          NativeModules.UIManager.AIRMap.Commands[name],
+          this._uiManagerCommand(name),
           args
         );
         break;
 
       case 'ios':
-        NativeModules.AIRMapManager[name].apply(
-          NativeModules.AIRMapManager[name],
+        this._mapManagerCommand(name).apply(
+          this._mapManagerCommand(name),
```

Comment:
This seems a bit fishy.  It calls the method with that same method that's being called as the `this` context?